client/src/App.tsx

```diff
@@ -1,8 +1,8 @@
-import React, { Component } from "react";
+import React from "react";
 import Web3 from 'web3';
+import { AbiItem } from 'web3-utils'
 import "./App.css";
 import { useEffect, useState } from 'react';
-import './App.css';
 import contract from './contracts/NFTCollection.json';
 
 function App() {
```

Comment:
sorry this is super basic - but I have no idea what that is/the syntax for typing here

Reply:
Oh I meant to highlight line `8` over there. You can type the `App` component with
```tsx
const App: React.FC = () => {
  return /** Some jsx*/
}
```
There is debate about this but I suggested it since you're making the box TS.